### ISSUE TITLE
When installing in Windows, fails to download Helm .zip file

### ISSUE THREAD

**Issue by saulimus** [IBM-Cloud/ibm-cloud-developer-tools#28]
**Summary**

**Steps to reproduce**
In PowerShell, run:
Set-ExecutionPolicy Unrestricted; iex(New-Object Net.WebClient).DownloadString('http://ibm.biz/idt-win-installer')
(as instructed at https://console.bluemix.net/docs/cloudnative/dev_cli.html#developercli)

**Operating System**
Windows 10

**Supporting details**
The binary file type has changed from .zip to .tar.gz for Helm 2.7.2, see:
https://github.com/kubernetes/helm/issues/3178

Install logs:
```
Checking for dependency git
git already installed
Checking for dependency docker
docker already installed
Checking for dependency kubectl
kubectl already installed
Checking for dependency helm
helm attempting to install...
Invoke-WebRequest : NoSuchKeyThe specified key does not exist.
At line:51 char:13
+             Invoke-WebRequest $helm_download_url -outfile "helm-$TAG- ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
mkdir : An item with the specified name C:\Program Files\helm already exists.
At line:52 char:13
+             mkdir "C:\Program Files\helm"
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceExists: (C:\Program Files\helm:String) [New-Item], IOException
    + FullyQualifiedErrorId : DirectoryExist,Microsoft.PowerShell.Commands.NewItemCommand
 
Expand-Archive : The path 'helm-v2.7.2-windows-amd64.zip' either does not exist or is not a valid file system path.
At line:53 char:13
+             Expand-Archive helm-$TAG-windows-amd64.zip -DestinationPa ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (helm-v2.7.2-windows-amd64.zip:String) [Expand-Archive], InvalidOperationException
    + FullyQualifiedErrorId : ArchiveCmdletPathNotFound,Expand-Archive
 
rm : Cannot find path 'C:\Users\sauls\helm-v2.7.2-windows-amd64.zip' because it does not exist.
At line:59 char:13
+             rm "helm-$TAG-windows-amd64.zip"
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\sauls\...ndows-amd64.zip:String) [Remove-Item], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.RemoveItemCommand
<etc ...>
```

**Workaround**
Manually download https://storage.googleapis.com/kubernetes-helm/helm-v2.7.2-windows-amd64.tar.gz and extract to C:\Program Files\helm

-----------------------------------

**Development "done" checklist**

- [ ] Test case to verify
- [ ] Public Documentation updated
- [ ] Change added to "release notes" as appropropriate
- [ ] Notification to stakeholders (OM, other squads, etc)

**Comment by kfbishop** [comment]
Thanks @saulimus.  We'll get this resolved soon.

**Comment by kfbishop** [comment]
resolved in v1.2
